plugins/main/bridge/bridge_test.go

```diff
@@ -1672,6 +1764,25 @@ var _ = Describe("bridge Operations", func() {
 		Expect(testutils.UnmountNS(targetNS)).To(Succeed())
 	})
 
+	var (
+		correctMinID int = 100
+		correctMaxID int = 105
+	)
+
+	DescribeTable(
+		"collectVlanTrunk succeeds",
+		func(vlanTrunks []*VlanTrunk, expectedVIDs []int) {
+			Expect(collectVlanTrunk(vlanTrunks)).To(Equal(expectedVIDs))
+		},
+		Entry("", &[]*VlanTrunk{}, []int{}),
+		Entry("", &[]*VlanTrunk{
```

Comment:
The type of vlanTrunk is wrong.

```suggestion
		Entry("", []*VlanTrunk{}, []int{}),
		Entry("", []*VlanTrunk{
```